packages/cogify/src/cogify/gdal.command.ts

```diff
@@ -53,45 +86,38 @@ export function gdalBuildVrtWarp(
 
 export function gdalBuildCog(targetTiff: URL, sourceVrt: URL, opt: CogifyCreationOptions): GdalCommand {
   const cfg = { ...Presets[opt.preset], ...opt };
-  const tileMatrix = TileMatrixSets.find(cfg.tileMatrix);
-  if (tileMatrix == null) throw new Error('Unable to find tileMatrix: ' + cfg.tileMatrix);
 
-  const bounds = tileMatrix.tileToSourceBounds(cfg.tile);
-  const tileExtent = [
-    Math.min(bounds.x, bounds.right),
-    Math.min(bounds.y, bounds.bottom),
-    Math.max(bounds.x, bounds.right),
-    Math.max(bounds.y, bounds.bottom),
-  ];
-
-  const targetResolution = tileMatrix.pixelScale(cfg.zoomLevel);
+  const targetOpts = getTargetOptions(cfg);
 
   return {
     command: 'gdal_translate',
     output: targetTiff,
     args: [
       ['-of', 'COG'],
+      cfg.srcwin ? ['-srcwin', cfg.srcwin[0], cfg.srcwin[1], cfg.srcwin[2], cfg.srcwin[3]] : undefined,
+      cfg.bigTIFF ? ['-co', `BIGTIFF=${cfg.bigTIFF}`] : ['-co', 'BIGTIFF=IF_NEEDED'], // BigTiff is somewhat slower and most (All?) of the COGS should be well below 4GB
```

Comment:
why do we need bigtiffs? everything should be well below 4gb?

Reply:
We set big tiff as `no` in the `topo-raster`, and `If_needed` for all others atm.